site/blog/red-team-ollama-model.md

```diff
@@ -31,31 +31,46 @@ ollama pull llama3.2
 
 ## Setting Up the Environment
 
+You can either initialize a new project or download the complete example:
+
+### Option 1: Download the Example
+
+```bash
+npx promptfoo@latest init --example redteam-ollama
+cd redteam-ollama
```

Comment:
Worth adding a line that mentions the location of the config

Also, ollama is supported in web ui so maybe we should prefer that?